cipd.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Configuration file to build cipd packages. 
+#
+# This file defines the cipd packages that are built from cocoon
+# It uses the following schema:
+#
+# targets:
+#   - target: <platform name> <project name>
+#     script: <name of build script> 
+
+targets:
+  - target: Mac codesign
```

Comment:
umm my understanding is that a CI.yaml target property will be parsed through the starlark file config/lib/ci_yaml/ci_yaml.star, and in starlark we might not have easy access to recipe functions such as cipd.build, cipd.register etc. 

Or maybe we are looking at parsing and building as part of cocoon/cocoon.py recipe, and make the cipd property part of tests.yaml. This way the logic is still part of recipe.

I might be wrong, but I think it would be easy to build through recipe than through starlark?

Reply:
Cocoon is on the cocoon scheduler, so the builds properties (api.properties.get) will have the latest ci.yaml property values for the commit

Reply:
Sounds good, I will add "script" properties for codesign and device_doctor targets, and retrieve them in cipd.py recipe through api.properties.get. Thanks for explaining!